scipy/stats/tests/test_stats.py

```diff
@@ -6883,6 +6883,73 @@ def test_dist_perm(self):
         assert_approx_equal(pvalue_dist, 0.001, significant=1)
 
 
+class TestConfInt:
+    """ Test the computation of non-parametric
+    confidence intervals for quantiles
+    """
+    np.random.seed(0)
+    m = 100                                     # number of tests
+    N = np.random.randint(2,100,size=m)         # num of samples
+    quantile = np.random.rand(m)                # quantile in [0,1)
+    confidence = 0.25*np.random.rand(m)+0.75    # confidence in [0.75,1)
+
+    def test_confint_naive(self):
+        # test against a naive implementation
+        m, N, quantile, confidence = self.m, self.N, self.quantile, self.confidence
+
+        def confint_naive(N,quantile,confidence):
+            '''
+            Naive implementation for computing the index of one-sided CI
+            (lower-bound) 
+            '''
+            # compute all probabilities from the binomiale distribution for the quantile of interest
+            bd=stats.binom(N,quantile)
+            ppm = [np.maximum(1-x,0.0) for x in np.cumsum([bd.pmf(k) for k in range(N)])]
```

Comment:
```suggestion
            ppm = [np.maximum(1-x, 0.0) for x in np.cumsum([bd.pmf(k) for k in range(N)])]
```
Please run `flake8` on these files to check PEP8.